packages/commonwealth/client/scripts/views/pages/view_proposal/index.tsx

```diff
@@ -64,8 +65,32 @@ const ViewProposalPage = ({
     };
   }, [proposal, forceRerender]);
 
+  const onAave = app.chain?.network === ChainNetwork.Aave;
+  const fetchAaveData = onAave && isAdapterLoaded;
+  const { data: cachedAaveProposals, isLoading: aaveProposalsLoading } =
+    useAaveProposalsQuery({
+      moduleReady: fetchAaveData,
+      chainId: app.chain?.id,
+    });
+
+  if (aaveProposalsLoading === false && fetchAaveData && !proposal) {
```

Comment:
`!aaveProposalsLoading` instead of `aaveProposalsLoading === false` should be fine?

Reply:
Updated.